packages/trader/src/AppV2/Containers/Trade/trade.tsx

```diff
@@ -30,21 +31,30 @@ const Trade = observer(() => {
     const {
         active_symbols,
         contract_type,
+        currency,
         has_cancellation,
         symbol,
         is_accumulator,
         is_market_closed,
         onMount,
         onChange,
+        onChangeMultiple,
         onUnmount,
+        setV2ParamsInitialValues,
     } = useTraderStore();
-    const { trade_types } = useContractsForCompany();
+    const { trade_types, available_contract_types } = useContractsForCompany();
     const [guide_dtrader_v2] = useLocalStorageData<Record<string, boolean>>('guide_dtrader_v2', {
         trade_types_selection: false,
         trade_page: false,
         positions_page: false,
     });
 
+    const is_crypto = isCryptocurrency(currency ?? '');
+    const default_stake = is_crypto
+        ? getMinPayout(currency ?? '')
```

Comment:
In trade-store I found logic for amount money conversion, based on currency. E.g. from prod: on demo account your default stake is 10 USD, but when you switch to crypto account like BTC, stake is changed to 0.00016 BTC. This logic for conversion is hardcoded on FE and currently used on production.
Here, I'm reusing it